Rewatch.lua

```diff
@@ -166,7 +166,7 @@ end
 -- display a debug message
 function Rewatch:Debug(message)
 
-	if(not rewatch.debug) then return end
+	--if(not rewatch.debug) then return end
```

Comment:
when done debugging, you may want to put this back in 😁

Reply:
Great, was really wondering how i enabled the debug messages :D